kadai1/imgconv/imgconv.go

```diff
@@ -0,0 +1,102 @@
+package imgconv
+
+import (
+	"fmt"
+	"image"
+	"image/jpeg"
+	"image/png"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+type Converter struct {
+	SrcExt string
+	DstExt string
+}
+
+var (
+	SupportExt = map[string]string{
+		"png":  ".png",
+		"jpg":  ".jpg",
+		"jpeg": ".jpg",
+	}
+	UnkownDecodeError = fmt.Errorf("Unkown Decode Error")
+	UnkownEncodeError = fmt.Errorf("Unkown Encode Error")
```

Comment:
> Error strings should not be capitalized (unless beginning with proper nouns or acronyms) or end with punctuation, since they are usually printed following other context.

see: https://github.com/golang/go/wiki/CodeReviewComments#error-strings

それと、エラー名は `ErrFoo`の形にしないとgolintさんに怒られます。

Reply:
golint入れてみます！ありがとうございます！